src/index.js

```diff
@@ -160,3 +162,16 @@ function fileEntry({ period, ref, norng }) {
     }
   }
 }
+
+// create a folder if it does not already exist
+function mkdirp(path, folderName) {
+  return cozyClient.files.statByPath(`${path}/${folderName}`).catch(err => {
+    log('info', err.message, `${path} folder does not exist yet, creating it`)
```

Comment:
Maybe we should make sure `err.code` is the right one (`ENOENT`?)? (no blocker)

Reply:
You are right, I will create another PR for this.